crates/bevy_sprite/src/mesh2d/mesh2d_view_types.wgsl

```diff
@@ -21,4 +21,8 @@ struct Globals {
     // Frame count since the start of the app.
     // It wraps to zero when it reaches the maximum value of a u32.
     frame_count: u32,
+#ifdef SIXTEEN_BYTE_ALIGNMENT
+    // WebGL2 structs must be 16 byte aligned.
+    _wasm_padding: f32
```

Comment:
Indeed, adding alignment attributes to the first or last (or middle!) members works.
```
// shader
#ifdef SIXTEEN_BYTE_ALIGNMENT
    @align(16)
#endif
    time: f32,

// rust
    #[cfg_attr(feature = "webgl", align(16))]
    time: f32,
```
Interestingly, as far as I can tell, this doesn't actually result in any additional padding in the generated glsl. So this seems to be more about appeasing wgpu rather than an actual webgl2 limitation?
```
// glsl (manual padding)
struct Globals {
    float time;
    float delta_time;
    uint frame_count;
    float _wasm_padding;
};

// glsl (alignment attributes) 
struct Globals {
    float time;
    float delta_time;
    uint frame_count;
};
```

I had previously attempted various alignment and padding attributes without luck (likely user error), but was lead to believe by https://github.com/gfx-rs/wgpu/issues/2832#issuecomment-1189942437 that manual padding was the way to go.

I'm in favor of this quick fix that is consistent with what we already have on the 3d side, but perhaps we should open an issue to track/investigate the issue further? It seems like pure coincidence that this is the only place where the issue shows up.

Reply:
I agree that this is worth investigating. Opened an issue here: #6685